src/types/session.types.ts

```diff
@@ -0,0 +1,17 @@
+export interface User {
+    iss: string;
+    sub: string;
+    role: string;
+    username: string;
+    email: string;
+    nickname: string;
+    jti: number;
+    nbf: number;
+    exp: number;
+}
+
+export interface Session {
```

Comment:
I don't think this is worth being a standalone interface as it probably is only used in useSession exports.